docs/user-guides/server.md

```diff
@@ -79,7 +79,7 @@ Please also note that **different models give different sizes of output dimensio
 | ViT-B-32::laion2b_e16                 | ✅       | ✅    | ✅        | 512              | 577             | 2.93                | 1.40                 |
 | ViT-B-32::laion400m_e31               | ✅       | ✅    | ✅        | 512              | 577             | 2.93                | 1.40                 |
 | ViT-B-32::laion400m_e32               | ✅       | ✅    | ✅        | 512              | 577             | 2.94                | 1.40                 |
-| ViT-B-32::laion2B-s34B-b79K           | ✅       | ✅    | ❌        | 512              | 577             | 2.94                | 1.40                 |
```

Comment:
Reject, the normalize is performed by the code, rather than using a normalized name.

Reply:
this follows https://github.com/mlfoundations/open_clip#pretrained-model-interface